src/acsets/schemas/catlab/StockFlowp.json

```diff
@@ -0,0 +1,91 @@
+{
+  "version": {
+    "ACSets": "0.0.0",
+    "ACSetSchema": "0.0.1",
+    "Catlab": "0.0.0"
+  },
+  "Ob": [
+    {
+      "name": "Flow",
+      "title": "Flow",
+      "description": null
+    },
+    {
+      "name": "Stock",
+      "title": "Stock",
+      "description": null
+    },
+    {
+      "name": "Link",
+      "title": "Link",
+      "description": null
+    }
+  ],
+  "Hom": [
+    {
+      "name": "u",
+      "codom": "Stock",
+      "dom": "Flow",
+      "title": "upstream",
+      "description": null
+    },
+    {
+      "name": "d",
+      "codom": "Stock",
+      "dom": "Flow",
+      "title": "downstream",
+      "description": null
+    },
+    {
+      "name": "s",
+      "codom": "Stock",
+      "dom": "Link",
+      "title": "source",
+      "description": null
+    },
+    {
+      "name": "t",
+      "codom": "Flow",
+      "dom": "Link",
+      "title": "target",
+      "description": null
+    }
+  ],
+  "AttrType": [
+    {
+      "name": "Name",
+      "title": "Names",
+      "description": "Name of an entity in the model.",
+      "ty": "str"
+    },
+    {
+      "name": "FuncFlow",
+      "title": "Flow Functions",
+      "description": "A representation of the flow function. Currently a math expression in string form.",
+      "ty": "str"
+    }
+  ],
+  "Attr": [
+    {
+      "title": "Stock Name",
+      "description": null,
+      "name": "sname",
+      "codom": "Name",
+      "dom": "Stock"
+    },
+    {
+      "title": "Flow Name",
+      "description": null,
+      "name": "fname",
+      "codom": "Name",
+      "dom": "Flow"
+    },
+    {
+      "title": "Flow Function",
+      "description": null,
+      "name": "ϕf",
```

Comment:
Are there any potential integration issues with using a less-standard unicode character as a JSON key?
I'm also wondering if this might be especially an issue since there are two unicode glyphs for phi, `U+03C6` and `U+03D5`.
While this is definitely the convention in Julia, perhaps for the interchange format it could be something that is potentially more universally accessible such as `ffunc` or `flowfunc`?

Reply:
We should test the JSON libraries. This would be a breaking change to StockFlow.jl so let's find an upstream bug before we patch it?

Reply:
Ah. I didn't realize that this would be a breaking change in the other library.

According to the JSON spec, it should be fine, and it obviously works in Python and JS, so I'm happy to assume that everything conforms to the spec until proven otherwise.

Reply:
Yeah the SF.jl library already exists and I was trying to get this in to py-acsets with the minimal code change possible. So far we haven't made any other changes to SF.jl.